src/script/lua_api/l_env.cpp

```diff
@@ -1342,7 +1342,7 @@ int ModApiEnv::l_find_path(lua_State *L)
 	return 0;
 }
 
-static bool read_tree_def(lua_State *L, int idx,
+bool ModApiEnvVM::read_tree_def(lua_State *L, int idx,
```

Comment:
should be moved to `common/c_converter.{cpp,h}`

Reply:
I moved to `c_content` and not `c_converter` since it's more fitting, I think.